README.md

```diff
@@ -79,4 +79,16 @@ end
 - Controller methods 
 - 
 
-
+## Testing Frontend
+- yarn add --dev jest babel-jest babel-preset-es2015
```

Comment:
I think you might have been able to get away with just installing jest without babel-jest and babel-preset-es2015